main_test.go

```diff
@@ -16,3 +16,29 @@ func TestShouldFail(t *testing.T) {
 		t.Fail()
 	}
 }
+
+func TestShouldFailNavInAnotherModule(t *testing.T) {
+	errors := check("fixture", "AnotherModule", "/modules/AnotherModule/nav.adoc")
+
+	if len(errors) == 1 {
+		error := errors[0]
+		if error != "one.adoc" {
+			t.Fail()
+		}
+	} else {
+		t.Fail()
+	}
```

Comment:
```suggestion
	assert.Len(t, errors, 1)
	assert.Contains(t, errors, "one.adoc")
```

See https://pkg.go.dev/github.com/stretchr/testify/assert.